secure_message/application.py

```diff
@@ -123,38 +123,41 @@ def get_client_token(client_id, client_secret, url):
 
     get_token_url = f'{url}/oauth/token'
 
+    s = requests.Session()
+    s.mount(get_token_url, HTTPAdapter(max_retries=15))
+
     try:
-        s = requests.Session()
-        s.mount(get_token_url, HTTPAdapter(max_retries=15))
-        response = requests.post(get_token_url,
-                                 headers=headers,
-                                 params=payload,
-                                 auth=(client_id, client_secret))
+        logger.debug("Attempting to GET client token from UAA")
+        response = s.post(get_token_url,
+                          headers=headers,
+                          params=payload,
+                          auth=(client_id, client_secret))
         response.raise_for_status()
+
+        try:
+            return response.json()
+            logger.debug("Decoding client token response json")
```

Comment:
These log statements will never be called

Reply:
fixed